### PR TITLE
[Map Rotation: Part 1] Map Standardisation

### PR DESCRIPTION
**Pull request by AffectedArc07**
## Remake Of #11099
This was remade because people wanted it. Part 2 will come after this has been testmerged, then merged in. I only tested basic functionality such as shuttle docking. I am requesting multiple testmerges, if possible on all 3 maps. Anyway, below is a copypaste of the previous PR
--------------------
# What does this PR do:
This PR is Phase 1 of my attempts to add map rotation into the game, without having to compile to change map. This PR standardises things between all the maps, so that the only difference between them is the actual station Z level. For a full list of changes, See the CL. 

This also adds a new variable to docking ports, `ignore_bounds`. If this is set to True, a shuttle will dock at a location regardless of if the docking port is too big or small. Why? Because the maps use different shuttles, and docking ports will complain if an area is too **BIG** for some reason. It makes no sense, dont ask me why. Please dont.

# Images of sprite/map changes:
**NOTE: These remaps are designed to be functional, not beautiful. They will be changed if I am pestered enough.**
MetaStation engineering shuttle dock:
Before:
image

After:
image
*A shuttle for the engineering outpost has been added*

MetaStation dorms:
Before:
image

After:
image
*The holodeck is now landscape, and the arcade has been moved*

DeltaStation engineering shuttle dock:
Before:
image

After:
image
*A shuttle for the engineering outpost has been added*

DeltaStation holodeck:
Before: 
image

After:
image
*The holodeck is now landscape*

Cyberiad AI sat before:
image

Cyberiad AI sat after:
image

*Replaced with the metastation sat. The cyberiad sat has a lot of wasted space, and no room for tcomms*

Delta AI sat before:
image

Delta AI sat after:
image
*Second AI upload has been replaced with telecomms*

# Changelog:
:cl: AffectedArc07
add: Z Level Standardisation. If you dont care about the inner workings of the maps, feel free to scroll to the next CL entry
add: DeltaStation now has the engineering outpost
add: MetaStation now has the engineering outpost
tweak: Cyberiad AI sat is now the MetaStation AI sat. This means that telecomms are now on the Cyberiad station instead of their own Z level
tweak: DeltaStation AI sat now has telecomms
tweak: DeltaStation holodeck is now landscape
tweak: MetaStation holodeck is now landscape
tweak: All maps now use the same Z-level layout, where the only different between maps now is the main station Z-level
del: Useless and duplicate Z-levels have been removed
tweak: Z-level file layout has been tweaked
tweak: Docking ports now have a variable to ignore landing boundaries. Use with caution.
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
And Appveyor & Travis cant pull BYOND to test compile with, becuase the skiddies have woken up

**Comment by AffectedArc07** [comment]
Unfucking travis

**Comment by AffectedArc07** [comment]
Will deconflict when asked

**Comment by AffectedArc07** [comment]
@Markolie those are old screenshots. The maps do use the shuttle chairs, as I asked myself _"Since when did para port those"_ when doing the maps.

**Comment by AffectedArc07** [comment]
@Markolie Im checking my local files now, but I assume they must be, because when I made this PR, I just re-created the edits on a latest master copy, not using my old branch.

**Comment by Kyep** [comment]
I am approving this, because:
- Having alternate maps change levels beyond z1 has always caused bugs when we use those alternate maps, as alternate versions of z2/etc tend not to be updated. This change lets us avoid an entire class of problem in future, and makes the use of alternate maps, even for testing, a lot more feasible.
- While it does change things on meta/delta/etc, frankly I don't much care about changes to those maps, as we never use them anyway, and even if the changes make them worse, that's a small price to pay for the map standardization that lets us use them in the first place.
- The only significant change to Box that I see in this PR is the movement of telecomms to being on the station zlevel, which, though it is a big change, I am neutral on. If you can break into the AI core, it seems reasonable you could break into telecomms.

**Comment by AffectedArc07** [comment]
Its just occured to me that I need somewhere for voidsuits to go

**Comment by TDSSS** [comment]
I'd say definitely move them, yeah. We don't want them to be a lot harder to get, IMO.

**Comment by AffectedArc07** [comment]
Things to do next time:
- Make the voidsuits spawn on the AI sat tcomms
- Check for all shuttles (Specops, SIT, SST, Nukeops, Whiteship, Ferry, Adminship, Mining ship, Engineering ship)
- Scream more

**Comment by AffectedArc07** [comment]
Deconflicted and ready for TM. Misc shuttles have been added to other maps. Mapdiffbot has all changes, but doesnt handle file renames well, and images may also take a bit to load, due to my internet being basically dial up.

**Comment by JKnutson101** [comment]
So the maps will all be loaded in on different Z-Levels? Since telecomms in now on AI Satellite, what about the rest of Z-Level 3(white ship, derelict teleporter)? Test Merge when? 👀

**Comment by AffectedArc07** [comment]
***insert some crying weeb gif or something***
image

**Comment by AffectedArc07** [comment]
If the maps conflict again I am actually going to cry

Also every map **SHOULD** be mapmerged now

**Comment by Couls** [comment]
image
:'(

**Comment by AffectedArc07** [comment]
I’m going to cry

**Comment by AffectedArc07** [comment]
Screw it I aint deconflicting till its about to be merged

**Comment by Cazdon** [comment]
You know the deletion of map files made me think this PR was *way* bigger than it was. Still massive but whew, that's a good that it's not actually 200,000 lines that need to be reviewed. (Only a massive 12,000 or so)

File: code/modules/shuttle/shuttle.dm [hunk 1]
```diff
@@ -154,7 +154,7 @@
 
 /obj/docking_port/stationary
 	name = "dock"
-
+	var/ignore_bounds = 0 // Do we ignore docking port clearances. USE WITH CAUTION!
```
Comment (farie82) [inline code/modules/shuttle/shuttle.dm:157]: FALSE

**Comment by AffectedArc07** [comment]
Todo: Standardise shuttles when I come back to this

**Comment by AffectedArc07** [comment]
This is being remade

**Pull request closed without merging**